includes/services/ArchiveService.php

```diff
@@ -1034,7 +1034,7 @@ protected function getSQLContent(string $privatePath): string
 
         $resultFile = tempnam($privatePath, 'tmp_sql_file_to_delete');
         $results = $this->consoleService->findAndStartExecutableSync(
-            "mysqldump",
+            "/usr/bin/mysqldump",
```

Comment:
```suggestion
            "/usr/bin/mysqldump",
```
pour des raisons de stabilité, je serais plutôt partisan de conserver `"mysqldump"` mais de remplacer plus bas `('\\' === DIRECTORY_SEPARATOR ? ["c:\\xampp\\mysql\\bin\\"] : []),` par `('\\' === DIRECTORY_SEPARATOR ? ["c:\\xampp\\mysql\\bin\\"] : ["/usr/bin/"]),`

Je vais faire un commit dans ce sens pour que le maximum de compatibilité reste présente

Reply:
oui, j'ai fait finalement `('\\' === DIRECTORY_SEPARATOR ? ["c:\\xampp\\mysql\\bin\\"] : ['/usr/bin', '/usr/local/bin']),` mais je te laisse commiter